Source/WebKit/NetworkProcess/curl/NetworkDataTaskCurl.h

```diff
@@ -45,16 +45,15 @@ namespace WebKit {
 
 class NetworkDataTaskCurl final : public NetworkDataTask, public WebCore::CurlRequestClient {
 public:
+    DECLARE_VIRTUAL_REFCOUNTED;
+
     static Ref<NetworkDataTask> create(NetworkSession& session, NetworkDataTaskClient& client, const NetworkLoadParameters& parameters)
     {
         return adoptRef(*new NetworkDataTaskCurl(session, client, parameters));
     }
 
     ~NetworkDataTaskCurl();
 
-    void ref() override { NetworkDataTask::ref(); }
-    void deref() override { NetworkDataTask::deref(); }
```

Comment:
You've replaced these effectively-final overrides (which would call NetworkDataTask's ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr's de/refs, not RefCounted's) with `DECLARE_VIRTUAL_REFCOUNTED`, I don't understand how this is equivalent. 🤔

Reply:
Oops! This is a bug.